gui/masspit.lua

```diff
@@ -0,0 +1,218 @@
+-- Interface to assign caged units in a stockpile to pond/pit.
+
+local utils = require 'utils'
+local gui = require('gui')
+local widgets = require('gui.widgets')
+
+Masspit = defclass(Masspit, widgets.Window)
+Masspit.ATTRS {
+    frame_title='masspit',
+    frame={w=43, h=22},
```

Comment:
Since you'll be having players interacting with the map, I'd suggest putting this window off to the side by default. How about `frame={w=43, h=22, r=2, t=18}`